app/src/main/java/com/example/easysurvey/dto/Survey.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.easysurvey.dto
+
+import com.google.gson.annotations.SerializedName
+
+data class Survey(@SerializedName("title")var title: String,
```

Comment:
created example data class Survey dto. Added example implementation of @SerializedName, which is used for deciphering the JSON names with the data class names.